docs/getting-started/devenv/get-started-with-wp-env.md

```diff
@@ -4,6 +4,8 @@ The [@wordpress/env](https://www.npmjs.com/package/@wordpress/env) package (`wp-
 
 Before following this guide, install [Node.js development tools](/docs/getting-started/devenv#node-js-development-tools) if you have not already done so.
 
+[![wp-env basics diagram](https://developer.wordpress.org/files/2023/10/wp-env-diagram.png)](https://excalidraw.com/#json=8Tp55B-R6Z6-pNGtmenU6,_DeBR1IBxuHNIKPTVEaseA)
```

Comment:
just to confirm the image will link to `https://excalidraw.com`?

Reply:
Yes

Reply:
I have separated the link to the original diagram from the image (now it's provided in the "Additional resources" section)